### ISSUE TITLE
Better Bandwidth capping

### ISSUE THREAD

**Issue by gregewing** [linuxserver/docker-nzbget#132]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Desired Behavior

The nzbget docker image should perform bandwidth capping more accurately than it currently does, and limit the bandwidth used, not just perform time-slicing to make bandwidth available for other systems.

## Current Behavior

The nzbget app appears to still use maximum bandwidth even though I have confgured it to use only 1Mbps (for example), it just only uses all available bandwidth some of the time instead of all the time.

## Alternatives Considered

I'd like to propose use if iproute tc command to perform bandwidth capping instead of using the in-built nzbget time-slicing method of bandwidth capping.

**Comment by aptalca** [comment]
This should be reported upstream to nzbget. We're not developing nzbget, we're only packaging it in a docker image.

**Comment by gregewing** [comment]
Yes I agree, however iproute tc doesn't work to manage bandwidth for a specific application, and it does not work from within a container, it has to be set up on the docker host.  I've produced a docker image for apt-get that writes the containers veth ID into a file in a volume, so that a cron script on the docker host can read it, then find the right docker adapter to apply the iproute traffic controls to.  

I'll log it with the nzbget developers too, but i'm not sure they will do much.  I've tried tthis with the apt-getdevelopment community already, though it is a sparse community, and got nowhere.
